pkg/rke2/rke2.go

```diff
@@ -77,7 +77,11 @@ func Agent(clx *cli.Context, cfg Config) error {
 }
 
 func setup(clx *cli.Context, cfg Config) error {
-	cisMode = clx.String("profile") == CISProfile
+	profile := clx.String("profile")
+	cisMode = profile == "cis-1.5" || profile == "cis-1.6"
+	if cisMode {
+		CISProfile = profile
```

Comment:
Why is this a variable now? Previously CISProfile was a constant so that we didn't have to have the string literal "cis-1.5" all over the place, but you've dropped that and turned it into a variable that is never used.

I think that you want to rename the CISProfile constant to CISProfile15, add a new CISProfile16 constant, and then replace all the string literals with those constants.